src/components/ServiceList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import FilterList from "../components/FilterList";
+import useServicesList from "../hooks/useServicesList";
+import { Card, CardContent } from "@baltimorecounty/dotgov-components";
+
+const ServiceList = () => {
+  const serviceItems = useServicesList();
+
+  const serviceCard = ({ name, department, url, icon, rank }) => (
+    <Card id={name}>
```

Comment:
Things that are still missing

- Most Popular Icon
- Most Popular Icon Accessibility message, see https://github.com/baltimorecounty/react-services-list/pull/5/files#diff-019a00e024a07fb5605b598fc002f4b8R53 (will have the class `sr-only`)

Reply:
done